### DISCUSSION TITLE
Upgrade from 2.2.7-pre 5 to v.2.4.15 or just wait until HACS provide upgrade alert?

### DISCUSSION THREAD

**Discussion by jprinsen** [WillCodeForCats/solaredge-modbus-multi#599] [Q&A]
Can somebody provide some guidance on how to update version 2.2.7-pre 5 to the newest version? Version 2.4.15 not showing up (yet?) in HACS whilst depreciation warnings being received in Logs. Advisable to wait until HACS provides upgrade alert?

Unfortunately Wiki is only referring to upgrade from pre-v2.2 versions). Thanks in anticipation,

Keep on doing this great work @WillCodeForCats and community - this integration is a great help!

**Comment by WillCodeForCats** [comment]
Releases are always available immediately through HACS, but I don't know why they wouldn't appear other than not matching the required HA core version.

**Reply by jprinsen** [reply]
Very odd. My HA Core is up-to-date to the latest version; same for HACS. Any other way to manually force an upgrade?

> Core 2024.5.3
> Supervisor 2024.05.1
> Operating System 
> Frontend 20240501.1
